sw/airborne/arch/chibios/mcu_periph/uart_arch.c

```diff
@@ -852,6 +862,14 @@ void uart_periph_set_baudrate(struct uart_periph *p, uint32_t baud )
 void uart_periph_set_mode(struct uart_periph *p __attribute__((unused)), bool tx_enabled __attribute__((unused)),
                           bool rx_enabled __attribute__((unused)), bool hw_flow_control __attribute__((unused))) {}
 
+#if defined STM32F7
+#define __USART_CR1_M USART_CR1_M_0
+#elif defined STM32F4
+#define __USART_CR1_M USART_CR1_M
+#else
+#error unsupported board
```

Comment:
I don't use F1 based board, so if you the correct parameters, I'll add them (maybe the same than F4 ?)